src/bitcoin-cli.cpp

```diff
@@ -85,7 +85,7 @@ static bool AppInitRPC(int argc, char* argv[])
         }
 
         fprintf(stdout, "%s", strUsage.c_str());
-        return false;
+        if(argc < 2) return false; else exit(0);
```

Comment:
I'm sure there's other solutions though. You just need more return values and handle them at the  call site. What about returning a `pair<bool,int>`?
- `<false, 1>` to exit error
- `<false, 0>` to exit success
- `<true,X>` to continue